raiden/network/proxies/token_network.py

```diff
@@ -1182,6 +1184,14 @@ def set_total_withdraw(
                     )
                     raise WithdrawMismatch(msg)
 
+                if expiration_block <= current_block_number:
+                    msg = (
+                        f"The current block number {current_block_number} is "
+                        f"already at expiration block {expiration_block} or "
+                        "later."
+                    )
+                    raise WithdrawMismatch(msg)
```

Comment:
The exception here should be `BrokenPreconditionError` rather than `WithdrawMismatch`

Reply:
I'll change this way.